### PR TITLE
Check Java properties when looking up RAILS_ENV

### PR DESCRIPTION
**Pull request by JasonLunn**
Adds support for #425

### CODE REVIEW

**Comment by JasonLunn** [comment]
@mkristian - something like this?

File: lib/warbler/templates/rails.erb [hunk 1]
```diff
@@ -1 +1 @@
-ENV['RAILS_ENV'] ||= '<%= config.webxml.rails.env %>'
+ENV['RAILS_ENV'] ||= Java::JavaLang::System.get_property( 'RAILS_ENV' ) || '<%= config.webxml.rails.env %>'
```
Comment (mkristian) [inline lib/warbler/templates/rails.erb]: please use `ENV_JAVA['RAILS_ENV']` instead

**Comment by JasonLunn** [comment]
Didn't even know about `ENV_JAVA` - thanks for the tip.

**Comment by JasonLunn** [comment]
What's the approximate timeline for a release that includes these changes?

**Comment by JasonLunn** [comment]
@mkristian ^^^

**Comment by mkristian** [comment]
@kares this question on the next release is more for you

**Pull request merged**